ui/src/shared/components/HistogramChart.tsx

```diff
@@ -122,6 +118,24 @@ class HistogramChart extends PureComponent<Props, State> {
     )
   }
 
+  private get xBrush(): JSX.Element {
+    const {onZoom} = this.props
+    const {xScale, adjustedWidth, adjustedHeight, bodyTransform} = this
+
+    if (onZoom) {
```

Comment:
Since the `onZoom` prop isn't used everywhere, let's just remove all this (i.e. the `XBrush` component and associated styles too).

Reply:
We left zoom in case the histogram gets used elsewhere and the drag to zoom functionality is desired